### PR TITLE
Gdscript doc

### PR DESCRIPTION
**Pull request by RaTi17**
Changed and added a bit of descriptions.
I left out fposmod as i'm not sure how to describe that one clearly.

### CODE REVIEW

**Comment by Noshyaar** [comment]
Changes look mostly okay. Please squash commits together.

File: doc/classes/@GDScript.xml [hunk 1]
```diff
@@ -778,14 +801,14 @@
 			<argument index="1" name="step" type="float">
 			</argument>
 			<description>
-				Snap float value to a given step.
+				Snaps float value "s" to a given "step".
```
Comment (Noshyaar) [inline doc/classes/@GDScript.xml]: Use [code] tag?

File: doc/classes/@GDScript.xml [hunk 2]
```diff
@@ -821,7 +844,7 @@
 			<argument index="0" name="var" type="Variant">
 			</argument>
 			<description>
-				Convert a Variant to json text.
+				Converts a Variant to json text.
```
Comment (Noshyaar) [inline doc/classes/@GDScript.xml]: `JSON` in uppercase?

**Comment by RaTi17** [comment]
Sorry about that, still getting a bit used to git usage.

**Comment by Noshyaar** [comment]
Don't worry. Thanks for your contribution! :)

File: doc/classes/@GDScript.xml (hunk 1 above)
Comment (RaTi17) [reply doc/classes/@GDScript.xml]: Like:
Snaps float value [code]"s"[/code] to a given [code]"step"[/code]
?

Comment (RaTi17) [reply doc/classes/@GDScript.xml]: About the previous example, it seems obvious now that the quotes should not be in that.

Comment (Noshyaar) [reply doc/classes/@GDScript.xml]: Yes, [code]s[/code] turns into `s`.

File: doc/classes/@GDScript.xml (hunk 2 above)
Comment (RaTi17) [reply doc/classes/@GDScript.xml]: Yes, you're right i'll fix that.

**Comment by RaTi17** [comment]
Hope this isn't a really stupid question but here it goes: i just updated my gdscript-doc branch on my own account, but i also see an immediate update here as a attached commit to your @Noshyaar last post. Is this supposed to happen ?

**Comment by Noshyaar** [comment]
Yes. This PR is basically a link to a branch in your repository. It says about this at the bottom of the comments.

`Add more commits by pushing to the gdscript-doc branch on RaTi17/godot.`

**Comment by RaTi17** [comment]
Thank you,  that clears that up, cool that it works that way.
Would it be advisable however to delete my local branch at some time ?
I wouldn't want to keep the 'link' alive forever (probably it gets closed when the pull request is closed but i'm not really sure how this works in actual practice).
Hope not to bother with too many questions, i promise this is the last one ;)

Really trying to also get the github interface, i just added a laugh icon, i think, to a one of the above posts while i meant more of a smile (though that one doesn't seem to exist). Meaning i started surrounding quoted parameters with code tags and then couldn't stop ;)

**Comment by NathanWarden** [comment]
@RaTi17 Once this pr is merged, a button will appear in this pr thread that you can press that will delete the branch. After that feel free to delete the local branch from your machine.

**Comment by RaTi17** [comment]
Thank you @Zireael07 and @NathanWarden, just learned something.
Cheers.

**Comment by willt** [comment]
This some of this was already done by this? https://github.com/godotengine/godot/pull/11227
Also did we decide to start using [code]param[/code] instead of 'param' ?

**Comment by RaTi17** [comment]
Yes they certainly overlap. I looked in the other xml files (for example in RayCast2D.xml) and code tags are being used, although not just for parameters but also lines of code.
Since this pr now conflicts, should i just close this request ?

**Comment by willt** [comment]
I would just update it based off the new code then re-push. if it's easier for you to close it and start over you can do that too.

**Comment by willt** [comment]
btw I did get confirmation we are use [code][/code] for params not single quotes now.

**Comment by RaTi17** [comment]
Not very proficient with git yet but i'll try to update this without closing the pr just yet. It's a nice practice anyway. If i can't get that to work i'll start a new one, thanks for the advice.

**Comment by akien-mga** [comment]
You can check http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html for a git howto focused on our workflow (and especially the last section about rebasing).

**Comment by RaTi17** [comment]
Thanks, i was already experimenting with resetting my branch, a bit enthusiastically i guess, I'll be sure to read that through.

**Pull request closed without merging**